sections/discussion/category-topic.tex

```diff
@@ -0,0 +1,46 @@
+\subsection{Category-topic model}\label{sec:discussion_category_topic} 
+Specific observations for the category-topic model can also be made.
+Just as for the author-topic model, the similarity between pairs of categories can be calculated.
+Because topic distributions are generated for each category, just as they are for each author in the author-topic model, \autoref{eq:author_similarity} can be used for categories as well, with $i$ and $j$ being the categories.
```

Comment:
```suggestion
Because topic distributions are generated for each category, they can also be calculated using \autoref{eq:author_similarity} where $i$ and $j$ are categories instead of authors.
```

Reply:
```suggestion
Because topic distributions are generated for each category, category similarity can also be calculated using \autoref{eq:author_similarity} where $i$ and $j$ are categories instead of authors.
```